### PR TITLE
MINOR: Web docs for KIP-220

### PR DESCRIPTION
**Pull request by guozhangwang**
1. added functions for KafkaStreams and KafkaClientSupplier.
2. added prefix for admin client in StreamsConfig.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by mikechengwei** [comment]
@guozhangwang   Can you tell me what tool to generate html ？  I only find docs folder including html file in project.

**Comment by guozhangwang** [comment]
@mikechengwei you can see this wiki for guidelines: https://cwiki.apache.org/confluence/display/KAFKA/Setup+Kafka+Website+on+Local+Apache+Server

**Comment by guozhangwang** [comment]
@mjsax @bbejeck @hachikuji please review.

**Comment by mikechengwei** [comment]
@guozhangwang I see the guidelines , and deploying website by apache server. But it not sloves my problem.  The display page is text style without css file rendering. 
I want to depoly pages the same with offical page .  Can you give me some advices？Thank you so much for helping me.

File: docs/streams/upgrade-guide.html [hunk 1]
```diff
@@ -44,6 +49,27 @@ <h1>Upgrade Guide &amp; API Changes</h1>
         See <a href="#streams_api_changes_0101">below</a> a complete list of 0.10.1 API changes that allow you to advance your application and/or simplify your code base, including the usage of new features.
     </p>
 
+    <h3><a id="streams_api_changes_110" href="#streams_api_changes_110">Streams API changes in 1.1.0</a></h3>
+
+    <p>
+        The introduction of <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-220%3A+Add+AdminClient+into+Kafka+Streams%27+ClientSupplier">KIP-220</a>
+        enables you to provide configuration parameters for the embedded admin client created by Kafka Streams, similar to the embedded producer and consumer clients.
+        You can provide the configs via the <code>StreamsConfig</code> by adding the configs with the prefix as defined by <code>StreamsConfig#adminClientPrefix(String)</code>
+        to distinguish them with configurations of other clients that share the same config names.
```
Comment (mjsax) [inline docs/streams/upgrade-guide.html]: `with` -> `from` ?

**Review by mjsax — COMMENTED** [review]
One minor comment (but I am not a native speaker...). Otherwise LGTM.

File: docs/streams/upgrade-guide.html [hunk 2]
```diff
@@ -44,6 +49,27 @@ <h1>Upgrade Guide &amp; API Changes</h1>
...
+    </p>
+
+    <p>
+        New methods in <code>KafkaStreams</code>:
+    </p>
+    <ul>
+        <li> added overloads for the constructor that allow to override the <code>Time</code> object used for tracking system wall-clock time; this is useful for unit testing your application code. </li>
```
Comment (bbejeck) [inline docs/streams/upgrade-guide.html]: `added overloads for the constructor that allow to override the`     -> `added overloads for the constructor that allows overriding the`

File: docs/streams/upgrade-guide.html [hunk 3]
```diff
@@ -44,6 +49,27 @@ <h1>Upgrade Guide &amp; API Changes</h1>
...
+    </ul>
+
+    <p> New methods in <code>KafkaClientSupplier</code>: </p>
+    <ul>
+        <li> added <code>#getAdminClient(config)</code> that allows to overwrite a <code>AdminClient</code> used for administrative requests such as internal topic creations, etc. </li>
```
Comment (bbejeck) [inline docs/streams/upgrade-guide.html]: `that allows to overwrite a`     -> `that allows overwriting an`

**Comment by bbejeck** [comment]
Left a couple of comments, otherwise LGTM

File: docs/streams/upgrade-guide.html [hunk 4]
```diff
@@ -20,6 +20,11 @@
 <script id="content-template" type="text/x-handlebars-template">
     <h1>Upgrade Guide &amp; API Changes</h1>
 
+    <p>
+        If you want to upgrade from 1.0.x to 1.1.0 you don't need to do any code changes as the public API is fully backward compatible.
```
Comment (hachikuji) [inline docs/streams/upgrade-guide.html]: "do" -> "make"?

File: docs/streams/upgrade-guide.html [hunk 5]
```diff
@@ -44,6 +49,27 @@ <h1>Upgrade Guide &amp; API Changes</h1>
         See <a href="#streams_api_changes_0101">below</a> a complete list of 0.10.1 API changes that allow you to advance your application and/or simplify your code base, including the usage of new features.
     </p>
 
+    <h3><a id="streams_api_changes_110" href="#streams_api_changes_110">Streams API changes in 1.1.0</a></h3>
+
+    <p>
+        The introduction of <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-220%3A+Add+AdminClient+into+Kafka+Streams%27+ClientSupplier">KIP-220</a>
+        enables you to provide configuration parameters for the embedded admin client created by Kafka Streams, similar to the embedded producer and consumer clients.
+        You can provide the configs via the <code>StreamsConfig</code> by adding the configs with the prefix as defined by <code>StreamsConfig#adminClientPrefix(String)</code>
```
Comment (hachikuji) [inline docs/streams/upgrade-guide.html]: "via the" -> "via"? Also, is it worth mentioning that the prefix is "admin."?

File: docs/streams/upgrade-guide.html [hunk 6]
```diff
@@ -44,6 +49,27 @@ <h1>Upgrade Guide &amp; API Changes</h1>
...
+        to distinguish them from configurations of other clients that share the same config names.
+    </p>
+
+    <p>
+        New methods in <code>KafkaStreams</code>:
+    </p>
+    <ul>
+        <li> added overload for the constructor that allows to override the <code>Time</code> object used for tracking system wall-clock time; this is useful for unit testing your application code. </li>
```
Comment (hachikuji) [inline docs/streams/upgrade-guide.html]: "allows to override" -> "allows overriding"?

File: docs/streams/upgrade-guide.html [hunk 7]
```diff
@@ -44,6 +49,27 @@ <h1>Upgrade Guide &amp; API Changes</h1>
...
+    </ul>
+
+    <p> New methods in <code>KafkaClientSupplier</code>: </p>
+    <ul>
+        <li> added <code>#getAdminClient(config)</code> that allows to overwrite an <code>AdminClient</code> used for administrative requests such as internal topic creations, etc. </li>
```
Comment (hachikuji) [inline docs/streams/upgrade-guide.html]: Do we need the `#` prefix? Also, do you mean "override"?

File: docs/streams/upgrade-guide.html [hunk 8]
```diff
@@ -20,6 +20,11 @@
 <script id="content-template" type="text/x-handlebars-template">
     <h1>Upgrade Guide &amp; API Changes</h1>
 
+    <p>
+        If you want to upgrade from 1.0.x to 1.1.0 you don't need to do any code changes as the public API is fully backward compatible.
+        See <a href="#streams_api_changes_110">below</a> a complete list of 1.1.0 API and semantic changes that allow you to advance your application with the usage of new features.
```
Comment (hachikuji) [inline docs/streams/upgrade-guide.html]: Missing "for" before "a complete list"? 

This sentence is a little wordy. Could we cut off "with the usage of new features"?

**Review by hachikuji — COMMENTED** [review]
Thanks, left a few minor comments.

File: docs/streams/upgrade-guide.html (hunk 7 above)
Comment (guozhangwang) [reply docs/streams/upgrade-guide.html]: Good point, I think we do not.

**Review by mjsax — APPROVED** [review]
LGTM

**Review by hachikuji — APPROVED** [review]
LGTM. @guozhangwang I'll leave you to merge.

**Comment by guozhangwang** [comment]
Merged to trunk.

**Pull request closed without merging**